File: TriviaServer/TriviaServer/CloseRoomResponse.cpp
#include "CloseRoomResponse.h"

CloseRoomResponse::CloseRoomResponse(int status)
{
	this->_status = status;
	this->messageCode = 53;
}

nlohmann::json CloseRoomResponse::castToJson() const
{
	return nlohmann::json{ {"status" , _status} };
}

File: TriviaWPFClient/TriviaClient/CloseRoomResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TriviaClient
{
    class CloseRoomResponse : Response
    {
        public const int MESSAGE_CODE = 53;

        public int status { set; get; }

        CloseRoomResponse() : base(MESSAGE_CODE) { }



    }
}
